api/handlers/videostream.go

```diff
@@ -159,12 +158,13 @@ func GetVideoStreams(ctx *fiber.Ctx) error {
 		return api.InvalidRequestURL(err)
 	}
 
-	// Validate timespan
-	if qry.TimeSpan != nil {
-		if qry.TimeSpan.Start.After(qry.TimeSpan.End) {
-			return api.InvalidRequestURL(errors.New("start time not before end time"))
-		}
-	}
+	// Validate timespan.
+	// TODO: Add validation directly to TimeSpan type.
+	// if qry.TimeSpan != nil {
```

Comment:
Can we remove this commented code if it's going to be validated elsewhere?
Also should this TODO be in the package with the type rather than here?

Reply:
Done :+1: